docs/JavaScript/README.md

```diff
@@ -132,3 +132,97 @@ Utility.handleHash();
 Utility.createElement('div');
 ...
 ```
+
+JSX Rules
+=============
+
+- Use functional stateless components (`FSC`).
```

Comment:
can we use useState and useEffect already?

Reply:
We have the React version that supports, you can try and see, and then we can revise the code style guide as they're in need of a revision anyway.